cmd/root_test.go

```diff
@@ -58,3 +61,43 @@ metadata:
 		t.Error("Expected parse fail, got success")
 	}
 }
+
+func TestPopulateSecretWithPasswords(t *testing.T) {
+	clientset := fake.NewSimpleClientset()
+	ns := "test"
+	id := "mysecret"
+	err := populateSecretWithPasswords(clientset, ns, id, []string{"pass1", "pass2"})
+	if err != nil {
+		t.Errorf("Unexpected error: %s", err)
+	}
+	secret, err := clientset.CoreV1().Secrets(ns).Get(id, metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("Unexpected error: %s", err)
+	}
+	if secret.Data["pass1"] == nil || len(secret.Data["pass1"]) != 10 {
+		t.Error("Failed to generate password")
+	}
+	if secret.Data["pass2"] == nil || len(secret.Data["pass2"]) != 10 {
+		t.Error("Failed to generate password")
+	}
```

Comment:
Check the two passwords aren't the same, just to make sure we haven't botched the random generation somehow:
```golang
if secret.Data["pass1"] == secret.Data["pass2"] {
   t.Errorf("Passwords are both %q", secret.Data["pass1"])
}
```